### PR TITLE
F#src 3749 unifying prefixes

### PR DESCRIPTION
**Pull request by mikramarc**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

File: sr_robot_launch/launch/sr_ur10arm_box.launch [hunk 1]
```diff
@@ -109,8 +109,8 @@
         <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
         <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
         <arg name="robot_description" value="$(arg robot_description)"/>
-        <arg name="hand_id" value="$(arg hand_id)"/>
-        <arg name="arm_id" value="$(arg arm_id)"/>
+        <arg name="hand_prefix" value="$(arg hand_prefix)"/>
+        <arg name="arm_id" value="$(arg arm_prefix)"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur10arm_box.launch]: Shouldn't this be change to arm_prefix as well?

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch [hunk 2]
```diff
@@ -89,7 +85,7 @@
     </group>
     <group if="$(arg grasp_controller)">
       <!-- Not available yet -->
-      <!--node name="$(arg hand_id)_grasp_controller" pkg="sr_utilities" type="grasp_controller_spawner.py" output="screen">
+      <!--node name="$(arg hand_prefix)_grasp_controller" pkg="sr_utilities" type="grasp_controller_spawner.py" output="screen">
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch]: should we leave this un-commented code here? in bimanual there isn't ever going to be a hand_prefix, right?

File: sr_robot_launch/launch/sr_ur10arm_box.launch [hunk 3]
```diff
@@ -40,10 +41,9 @@
   <arg name="joint_state_pub_frequency" default="125"/>
 
   <!-- HAND CONFIG-->
-  <arg name="hand_id" default="H1"/>
+  <arg name="hand_prefix" default="H1"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur10arm_box.launch]: H1?

File: sr_robot_launch/launch/sr_ur10arm_box.launch [hunk 4]
```diff
@@ -109,8 +109,8 @@
         <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
         <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
         <arg name="robot_description" value="$(arg robot_description)"/>
-        <arg name="hand_id" value="$(arg hand_id)"/>
-        <arg name="arm_id" value="$(arg arm_id)"/>
+        <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur10arm_box.launch]: I don't think we need to pass hand_prefix or arm_prefix, right, just side

File: sr_robot_launch/launch/sr_ur3arm_box.launch [hunk 5]
```diff
@@ -31,10 +31,10 @@
   <arg name="joint_state_pub_frequency" default="125"/>
 
   <!-- HAND CONFIG-->
-  <arg name="hand_id" default="H1"/>
+  <arg name="hand_prefix" default="H1"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur3arm_box.launch]: h1?

File: sr_robot_launch/launch/sr_ur3arm_box.launch [hunk 6]
```diff
@@ -104,8 +104,8 @@
         <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
         <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
         <arg name="robot_description" value="$(arg robot_description)"/>
-        <arg name="hand_id" value="$(arg hand_id)"/>
-        <arg name="arm_id" value="$(arg arm_id)"/>
+        <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur3arm_box.launch]: again I think we only needto pass side here

File: sr_robot_launch/launch/sr_ur5arm_box.launch [hunk 7]
```diff
@@ -31,10 +31,10 @@
   <arg name="joint_state_pub_frequency" default="125"/>
 
   <!-- HAND CONFIG-->
-  <arg name="hand_id" default="H1"/>
+  <arg name="hand_prefix" default="H1"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur5arm_box.launch]: H1?

File: sr_robot_launch/launch/sr_ur5arm_box.launch [hunk 8]
```diff
@@ -31,10 +31,10 @@
...
 
   <!-- ARM CONFIG-->
-  <arg name="arm_id" default="ra"/>
+  <arg name="arm_prefix" default="ra"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur5arm_box.launch:34]: why not have arm_prefix take the first letter of side as you do in other launch files?

File: sr_robot_launch/launch/sr_ur5arm_box.launch [hunk 9]
```diff
@@ -104,8 +104,8 @@
         <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
         <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
         <arg name="robot_description" value="$(arg robot_description)"/>
-        <arg name="hand_id" value="$(arg hand_id)"/>
-        <arg name="arm_id" value="$(arg arm_id)"/>
+        <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur5arm_box.launch]: I think we can just pass 'side' here

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 10]
```diff
@@ -138,8 +140,8 @@
         <arg name="debug" value="$(arg debug)"/>
         <arg name="initial_z" value="$(arg initial_z)"/>
         <arg name="robot_description" value="$(arg robot_description)"/>
-        <arg name="hand_id" value="$(arg hand_id)"/>
-        <arg name="arm_id" value="$(arg arm_id)"/>
+        <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: same thing again with only sending 'side'

File: sr_robot_launch/launch/sr_ur_arms_hand.launch [hunk 11]
```diff
@@ -149,7 +150,7 @@
       <arg name="robot_description" value="$(arg robot_description)"/>
       <arg name="pwm_control" value="$(arg pwm_control)"/>
       <arg name="hand_serial" value="$(arg hand_serial)"/>
-      <arg name="hand_id" value="$(arg hand_id)"/>
+      <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_ur_arms_hand.launch]: I wonder if it is always better to just send side and have the other prefixes based on that?

File: sr_robot_launch/launch/srhand.launch [hunk 12]
```diff
@@ -1,6 +1,9 @@
 <launch>
+  <!--This parameter is deprecated. It will be removed soon -->
+  <arg name="hand_id" default=""/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/srhand.launch]: why not just replace hand_id with hand_prefix?

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**